edb/testbase/server.py

```diff
@@ -1838,7 +1841,7 @@ def get_test_cases_setup(
         except unittest.SkipTest:
             continue
 
-        if not setup_script:
+        if setup_script is None:
```

Comment:
Giving `''` special treatment gives me heebie-jeebies.  Also is this really ever `None` currently? Maybe that's what we should be returning instead?

Reply:
Yeah, sure

Reply:
I'll drop the special handling of `''` but its best to keep returning that instead of `None`, since some subclasses want to concatenate stuff